articles/110_interface_definition.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: default
+title: Interface definition
+permalink: articles/interface_definition.html
+abstract:
+  This article specifies the format to specify the data structures ROS 2 components exchange to interact with each other.
```

Comment:
this sentence is hard to read, maybe:

"This article specifies the format and data structures exchanged by ROS 2 components to interact with each other."